integration-tests/rocketgate_test.go

```diff
@@ -0,0 +1,2 @@
+package test
```

Comment:
Adding tests in a later commit?

Reply:
Yeah, @rlcooper46 is working on it and will push here.

Reply:
He has put in https://github.com/BoltApp/sleet/pull/194/files